desk/tests/app/logs.hoon

```diff
@@ -0,0 +1,24 @@
+/-  l=logs
+/+  *test-agent
+/=  agent  /app/logs
+|%
+++  dap  %logs-test
+++  test-log-fail
+  %-  eval-mare
+  =/  m  (mare ,~)
+  ;<  *  bind:m  (do-init dap agent)
+  :: ;<  *  bind:m  (set-scry-gate ,~)
+  ;<  *  bind:m  (jab-bowl |=(b=bowl b(sap /gall/test)))
+  ;<  =bowl:gall  bind:m  get-bowl
+  =/  ev-fail=log-event:l
+    [%fail %test-fail leaf+"test stacktrace" ~]
+  ;<  caz=(list card)  bind:m
+    (do-poke log-action+!>([%log ev-fail]))
+  =/  fard=(fyrd:khan cage)
+    [q.byk.bowl %posthog noun+!>(`[%test ^-(log-item:l [now.bowl ev-fail])])]
+  ::  expect log submission -posthog
+  ::
+  %+  ex-cards  caz
+  :~  (ex-arvo /posthog/test/(scot %da now.bowl) %k %fard fard)
+  ==
```

Comment:
As discussed out of band, the fact that the card here contains a vase is troublesome, since this logic as written ends up doing a noun comparison, which isn't sane for types (their contexts might be different even if they otherwise nest).

Short-term you probably just want to test the card explicitly here, unpacking it as you go, and then doing an `+expect-eq:test` on the vase it contains.

Longer-term, it might be good for `/lib/test-agent`'s card comparison functions to know where vases might be, and do an extraction internally, just like `+ex-fact` and `+ex-poke` already do. That's out of scope for here though.